options_compile.cpp

```diff
@@ -287,10 +287,11 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
 
   auto parseClExt = [&](const std::string &clExtStr) {
     llvm::StringRef clExtRef(clExtStr);
-    clExtRef.consume_front("-cl-ext=");
+    if (!clExtRef.consume_front("-cl-ext="))
```

Comment:
Assert return value is true? The use place of parseClExt requires a starts with "-cl-ext="

Reply:
done, thanks.